File: Dante1/Module1.01/Zadanie1.19/main.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    float num;

    printf("Enter number:");

    int check = scanf("%f",&num);
    if(check != 1)
    {
        return 1;
    }

    printf("Result is = %f",2 * num * num * num - 4 * num * num + 3 * num - 7);
    return 0;
}

File: Dante1/Module1.01/Zadanie1.19/main.rs
fn main() 
{
    println!("Enter x: ");
    
    let mut x = String::new();
    std::io::stdin().read_line(&mut x).expect("Failed to read line");
    
    let x: f64 = x.trim().parse().expect("Failed to parse into a number");
    
    println!("Result is: {:.6}", 2.0 * x.powi(3) - 4.0 * x.powi(2) + 3.0 * x - 7.0);
    
    return;
}
